helm/alfresco-content-services/values.yaml

```diff
@@ -525,25 +525,39 @@ share:
 # -- Defines the mounting points for the persistence required by the apps in the
 # cluster the alf_data folder from alfresco-content-repository app is mapped to
 # alfresco-content-services/repository-data
+#persistence:
+#  enabled: true
+#  baseSize: 20Gi
+#  # -- Enable and define if you already have a custom storage class defined
+#  storageClass:
+#    enabled: false
+#    accessModes:
+#      - ReadWriteMany
+#    # -- Custom storage class name
+#    name: "nfs"
+#  existingClaim: "alfresco-volume-claim"
+#  repository:
+#   enabled: true
+#    config:
+#      querysetsMountPath: "/usr/local/tomcat/shared/classes/alfresco/extension/querysets/"
+#      transform:
+#        renditionsMountPath: "/usr/local/tomcat/shared/classes/alfresco/extension/transform/renditions/"
+#        pipelinesMountPath: "/usr/local/tomcat/shared/classes/alfresco/extension/transform/pipelines/"
+#        mimetypesMountPath: "/usr/local/tomcat/shared/classes/alfresco/extension/mimetypes/"
+#    data:
+#      mountPath: "/usr/local/tomcat/alf_data"
+#      subPath: "alfresco-content-services/repository-data"
+
 persistence:
```

Comment:
there should not be any root `persistence` data structure after this PR.

Reply:
fixed

Reply:
that means we expect a removal here